### DISCUSSION TITLE
Maple syrup?

### DISCUSSION THREAD

**Discussion by Envii2** [PhantomWing/RusticDelight#11] [Ideas]
Since pancakes are a part of this mod, why not go a step further and add maple syrup to use on the pancakes? It could be done by adding a new tree (maple tree) or choosing a pre-existing tree that can have sap harvested from it and turned into syrup.

**Comment by PhantomWing** [comment]
Definitely a good idea that suits the mod well! It could work like some kind of tapper (as in Stardew Valley). I don't want to add stuff like custom trees though, for better compatibility with world generation mods. I will think about it some more, thanks for the suggestion :)

**Reply by VaderViktor** [reply]
If you aren't interested in adding a new tree type (and therefore new woodtype and assosiated items), you could have syrup be an item obtained through villager trades or through the wandering trader. Maybe the trader sells a keg of it, which you can take bottles out as if it were a cauldron?
Alternatively, Autumnity by the Abnormals team has maple trees and maple syrup, and you could integrate compatibility with that mod. They also add pancakes though, so I'm not entirely sure how much you'd want to acknowledge and integrate into it.

One thing I'd like to add just in general, make sure to avoid bloat as you move forward. the best thing about this mod is that is is concise and there doesnt seem to be any pointless items. Another addon, Fruits Delight, ive recently removed from my modlist on my new playthrough for this reason. The several fruits that all behave the same, and the new trees that just swap wood type and leaf type, it just felt subpar i guess, and thats something I hope you avoid as you continue to develop the mod.

**Reply by PhantomWing** [reply]
Good suggestion about the Syrup! I'm now thinking to have Syrup be cooked by mixing honey/sugar and water, and also available as a trade. And then add compatibility with syrups from other mods, that add stuff like tree tapping!

Definitely agree about the bloat. A lot of mods add lots of items/crops that behave kind of the same. I like my mod to keep mechanics simple and concise, and stay true to that core cozy Minecraft and Farmer's Delight vibe. For example for the coffee, I wanted a mechanic that just used the Cooking Pot, instead of adding more utility blocks like some other mods. It makes things unnecessarily complex in my opinion. And I try to make every crop feel unique, so it doesn't feel like copy-paste haha!

**Comment by PhantomWing** [comment]
The newest Rustic Delight update (1.4) has added Syrup! Use it to craft two new types of pancakes: Pancakes (the classic variant), and Pumpkin Pancakes! In addition, the Syrup acts as a cooking ingredient for other food items, such as a Syrup Cookie, Syrup Cheesecake and Syrup Sandwich. Hope you like it!
